sources/commands/materialize.ts

```diff
@@ -8,20 +8,19 @@ import {
 } from "@yarnpkg/core";
 import { BaseCommand } from "@yarnpkg/cli";
 import { Command, Option, Usage } from "clipanion";
-import * as t from "typanion";
+import * as t from "typanion-custom";
```

Comment:
Avoid using older packages that come with `yarn`.

Reply:
Does the new version contain a bugfix that we need?

Reply:
Seems not, as this was included in `typanion` released two days ago. And the current `yarn` has a regression bug and we cannot upgrade.

Reply:
https://github.com/nicolo-ribaudo/yarn-plugin-conditions/pull/3/files#diff-4870593b944b1bec1926bc12777af7fe7a3df32202cda3614956ac82bc31b89fL40-R44

Maybe I restore this?